kuma/wiki/apps.py

```diff
@@ -146,4 +152,4 @@ def on_document_spam_attempt_save(self, sender, instance, **kwargs):
         body = render_to_string('wiki/email/spam.ltxt',
                                 {'spam_attempt': instance})
         send_mail(subject, body, settings.DEFAULT_FROM_EMAIL,
-                  ['mdn-spam-watch@mozilla.com'])
+                  [config.EMAIL_LIST_FOR_FIRST_EDITS])
```

Comment:
The default and current value of ``EMAIL_LIST_FOR_FIRST_EDITS`` is ``mdn-spam-watch@mozilla.com``. Adding another commit to make the switch to ``EMAIL_LIST_SPAM_WATCH``...